src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -210,9 +210,9 @@ export default {
 	},
 
 	watch: {
-		showDeviceChecker(newValue) {
+		showMediaSettings(newValue) {
 			const browserValue = newValue ? 'true' : 'false'
-			BrowserStorage.setItem('showDeviceChecker' + this.token, browserValue)
+			BrowserStorage.setItem('showMediaSettings' + this.token, browserValue)
```

Comment:
**Must:** This resets basically the behaviour and all users that opted out of it will see it again. Is it needed?
```suggestion
			BrowserStorage.setItem('showDeviceChecker' + this.token, browserValue)
```

Reply:
I think it's totally fine, and there's a new nice feature that they might want to check out